scripts/amd/gemm/tune_gemm.py

```diff
@@ -202,13 +221,14 @@ def matmul_{configStr}(a, b, c, M, N, K, am, ak, bk, bn, cm, cn, warmup=False):
             num_stages = {num_stages},
             waves_per_eu = {waves_per_eu},
             matrix_instr_nonkdim = {mfmaInstrSize},
-            kpack = {kpack}
+            kpack = {kpack},
+            BIAS={use_bias},
```

Comment:
add space around `=` so all args have the same format?

Reply:
fixed.